tests/disruption_helpers.py

```diff
@@ -0,0 +1,56 @@
+import logging
+
+from resources import pod
+from ocs import constants, ocp
+from ocsci import config
+logger = logging.getLogger(__name__)
+
+POD = ocp.OCP(kind=constants.POD, namespace=config.ENV_DATA['cluster_namespace'])
+
+
+class Disruptions:
+    """
+    TBD
+    """
+    def get_mgr(self):
+        self.mgr_pod = pod.get_ceph_mgr_pod()
+
+    def get_mon(self):
+        self.mons = pod.get_ceph_mon_pods()
+
+    def get_mds(self):
+        self.mds_pods = pod.get_ceph_mon_pods()
+
+    def get_osd(self):
+        self.osd_pods = pod.get_ceph_mon_pods()
+
+    def delete_mgr(self):
+        self.mgr_pod.delete()
```

Comment:
Is your intention to shutdown the pods gracefully ? if Yes then this is correct else we might have have to figure out how to perform SIGKILL for containers which would be really destructive.

Reply:
@shylesh do you mean killing the MON service from within the MON pod ?

In MON and MGR pods, the respective MON and MGR services couldn't be killed(PID was 1).  Hence, we had to stick with delete(resulting in respin) of these pods.
In OSDs, we were able to kill the OSD services. 

Need to test these in latest Ceph-rook-csi builds in case there have been any changes.

Reply:
@nehaberry ,
I meant SIGKILL for containers from outside. The problem with SIGTERM which is graceful termination is, processes will get a chance to clearly flush pending buffers and chances of catching something getting corrupted (this might what happen in real world) will be low. Current way of deleting resources may provide some service interruptions but was that  the intention ? if Yes then this is fine.

Reply:
> @shylesh @nehaberry I'm open to suggestions. We should go with the way which is the most destructive and close to real world as possible.
> I suggest we keep it with delete() for now and I'll open an issue to provide a way to perform SIGKILL for containers from outside and maybe change the implementation here to SIGKILL when it'll be available.

Yep , makes sense.

Reply:
https://github.com/red-hat-storage/ocs-ci/issues/303